src/dotnet-uninstall/Shared/SdkInfo/SdkVersion.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.SdkInfo
+{
+    class SdkVersion : IEquatable<SdkVersion>, IComparable<SdkVersion>, IComparable
```

Comment:
I like this class a lot.

nit: add explicit internal

Reply:
I'm just wondering why explicit `internal`s are preferred, cus I just removed all `internal`s to make code clearer.

Reply:
Will fix that. [Discussed]

Reply:
done.